android/src/main/java/com/tailscale/ipn/ui/viewModel/IpnViewModel.kt

```diff
@@ -85,14 +85,21 @@ open class IpnViewModel : ViewModel() {
 
   // Login/Logout
 
-  fun login(options: Ipn.Options = Ipn.Options(), completionHandler: (Result<Unit>) -> Unit = {}) {
-
+  fun login(
+      options: Ipn.Options = Ipn.Options(),
```

Comment:
maybe you should just take the authKey string only? You don't have separate frontend-vs-backend log IDs on Android now, do you?

Reply:
Good call.